src/recipes/heartbeat-by-user-first-impression/index.js

```diff
@@ -161,11 +150,11 @@ let run = function (state, extras) {
   eData.store();
 
   let locale = (state.locale || "UNK").toLowerCase();
-  let trans = translations[locale] || {};
+  let trans = translations[locale] || translations['en-us']; //
```

Comment:
Perhaps export a function in src/localeStrings/index.js instead?  That way we could handle default resolution there.